CMakeLists.txt

```diff
@@ -130,6 +130,7 @@ set(CPU_SOURCES
   test/test_regression.cpp
```

Comment:
@miquelflorensa Could you please comment lines 7, 8, and 9 and uncomment lines 4, 5, and 6 in [builld_wheels.yml](https://github.com/lhnguyen102/cuTAGI/blob/main/.github/workflows/build_wheels.yml) before merging. We don't want to run all builds if any PR is submitted because the number of minutes for git actions is limited